images/check-trigger-trusted-apps/Dockerfile

```
@@ -0,0 +1,5 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+LABEL maintainer="jguzik@redhat.com"
+
+ADD check-trigger-trusted-apps /usr/bin/check-trigger-trusted-apps
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)